source/version.js

```diff
@@ -1,31 +1,57 @@
 'use strict';
 const semver = require('semver');
 
-exports.SEMVER_INCREMENTS = ['patch', 'minor', 'major', 'prepatch', 'preminor', 'premajor', 'prerelease'];
-exports.PRERELEASE_VERSIONS = ['prepatch', 'preminor', 'premajor', 'prerelease'];
+class Version {
+	constructor(version) {
+		this.version = version;
```

Comment:
I just noticed [`semver.coerce`](https://github.com/npm/node-semver#coercion) exists, which seems like a better option than just using the given version. However, it would break `isPrerelease`, which should probably  be split up to two methods anyways: `isPrerelease` and `isPrereleaseBump` (the first would stay in the class and the second would become a utility method outside of it)).

Reply:
👍